src/components/Map.js

```diff
@@ -10,17 +14,25 @@ function Map(){
         zoom: 11
 
     };
+
+    useEffect(()=> {
+        fetchNearbyPlaces(defaultLocation.center.lat, defaultLocation.center.lng).then(res => setNearbyPlaces(res));
+    }, []);
+    
     return (
         <div style={{ height: 'calc(66.67vh - 1.25rem)', width: '100%' }}>
           <GoogleMapReact
+            distanceToMouse={() => null }
```

Comment:
I had to look this up! 😅 What's it being used for?

Reply:
It was related to this issue here with the package:

[X of undefined on mouse enter / move on map](https://github.com/google-map-react/google-map-react/issues/677)

When trying to fix this issue, we did not consider removing React.StrictMode, so we used one of the other suggested changes of setting distanceToMouse prop to be null. Now that we have removed React.StrictMode, the error does not pop up regardless of setting that prop to null. 

@segdeha should I remove it before merging to master?

Reply:
Yep, if it's not needed!

Reply:
Fixed in latest commit!